.env.template

```
@@ -30,3 +30,7 @@ MYSQL_PASSWORD=password
 # GITHUB
 GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
+
+# AWS
+UNIX_OPS_SHARED_SECRET=
+AUTOLAB_HOST_COURSES_ROOT=/home/ubuntu/autolab-docker/Autolab/courses
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTOLAB_HOST_COURSES_ROOT key should go before the UNIX_OPS_SHARED_SECRET key

(UnorderedKey)